TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -470,9 +470,14 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
 
     private func buyingPowerText() -> String? {
         guard let buyingPower = self.order.linkedBrokerAccount?.balance?.buyingPower else { return nil }
-        return "Buying Power: " + NumberFormatter.formatCurrency(
-            buyingPower,
-            currencyCode: self.order.linkedBrokerAccount?.accountBaseCurrency
+        let buyingPowerLabel = self.order.linkedBrokerAccount?.balance?.buyingPowerLabel ?? "Buying Power"
+        return (
```

Comment:
Might be clearer to pull the `formatCurrency` call out to a variable and then use string interpolation